helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -211,5 +211,13 @@
     <record id="helpdesk_ticket_channel_other" model="helpdesk.ticket.channel">
       <field name="name">Other</field>
     </record>
+    <!--default alias for leads-->
```

Comment:
Leads? Why is this in data and not in demo? There shouldn't be any "default" alias, and more if empty.

Reply:
You are right... this record not has sense (on CRM use it to compute a default alias for leads, but here we do not need anything similar)

Reply:
Done